### ISSUE TITLE
Collapsible panel: No keyboard accessibility

### ISSUE THREAD

**Issue by ccweb** [swiss/styleguide#497]
Collapsible panel does not get the focus, when accessing the page via keyboard.
Tested on page:
https://swiss.github.io/styleguide/en/content-containers.html#02-content-containers-03-collapsible-panel
1. Navigate through page by pressing the tab key.
2. The Link «Collapsible panel» gets the focus and one tab keystroke later the </> Link gets the focus
![collapsible-panel](https://cloud.githubusercontent.com/assets/7446943/20428059/6793d12a-ad87-11e6-9006-d84a23bcef34.png)

**Comment by bit-pro-iew-eui** [comment]
Reproduced issue on IE11, the second collapsible group is either not accessible through the keyboard. In Chrome the behaviour is different.

**Comment by ccweb** [comment]
This problem is still existing. The first collapsible panel (Collapsible Group Item #1) is accessible by keyboard. The second (Collapsible Group Item #2) however never.
![14-11-_2017_17-00-42](https://user-images.githubusercontent.com/7446943/32790003-7475f196-c95d-11e7-879c-aca17d8003d4.png)
Tested on page https://swiss.github.io/styleguide/en/content-containers.html#02-content-containers-03-collapsible-panel in IE 11 (Version 11.0.6900.18816)

**Comment by gillerr** [comment]
The solution to this issue will be implemented in issue #598
